test/unit/instance-validator.test.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+/* jshint -W030 */
+const chai = require('chai');
+const expect = chai.expect;
+const Support = require(__dirname + '/support');
+const InstanceValidator = require('../../lib/instance-validator');
+const sinon = require('sinon');
+var Promise = Support.sequelize.Promise;
+const SequelizeValidationError = require('../../lib/errors').ValidationError;
+
+describe(Support.getTestDialectTeaser('InstanceValidator'), () => {
+
+  it('configures itself to run hooks by default', () => {
+    const instanceValidator = new InstanceValidator();
+    expect(instanceValidator.options.hooks).to.equal(true);
+  });
+
+  beforeEach(() => {
```

Comment:
Good practice to keep `before/after [Each]` on top of the test, easy to see what this test suite depends upon